src/main/java/com/team2/hrbank/changelog/dto/EmployeeLogDto.java

```diff
@@ -11,49 +11,56 @@ private EmployeeLogDto() {
     }
 
     @Builder
-    public record EmployeeBaseInfo(
+    public record CreateLogRequest(
+            // Employee info
             LocalDateTime hireDate,
             String name,
             String position,
             String department,
             String email,
             String employeeNumber,
-            String status
-    ) {}
-
-    public record LogMetadata(
-            @Nullable
-            String memo,
+            String status,
+            // metadata
+            @Nullable String memo,
             String ipAddress
-    ) {
-
-        public static LogMetadata withMemo(String ipAddress, String memo) {
-            return new LogMetadata(memo, ipAddress);
-        }
-
-        public static LogMetadata withoutMemo(String ipAddress) {
-            return new LogMetadata(null, ipAddress);
-        }
-
-    }
-
-    @Builder
-    public record CreateLogRequest(
-            EmployeeBaseInfo employeeInfo,
-            LogMetadata metadata
     ) {}
```

Comment:
Update와 일관성 있게 변경한 코드있니다.

```suggestion
    public record CreateLogRequest(
        String employeeNumber, // 대상 직원 사번
        @Nullable String memo, // 메모
        
        // 변경 상세 내용
        String employeeName, // 직원 이름
        String employeeEmail, // 직원 이메일
        String departmentName, // 부서 이름
        String employeePosition, // 직원 직함
        LocalDate employeeHireDate, // 직원 입사일
        String employeeStatus, // 직원 상태
        
        String ipAddress, // IP 주소
        LocalDateTime at // 시간
    ) {}
```